### ISSUE TITLE
RTL headline block

### ISSUE THREAD

**Issue by twinlensreflex** [bbc/simorgh#1799]
**Is your feature request related to a problem? Please describe.**
We want our headline block to work for RTL articles (Persian, Arabic, Urdu, Pashto).

**Describe the solution you'd like**
As a result of https://github.com/bbc/simorgh/issues/2713, the NassimPersian font should be loaded for all 4 services and we should already have the text direction being right-to-left. This ticket is to ensure that the implementation matches the designs (except for it not using the correct service-specific font).

If the font sizes and line heights need updating, they should be discussed with UX design before implementing any changes `gel-foundations`

For each service we should import the Arabic typography settings from here: `import { arabic } from '@bbc/gel-foundations/scripts'`

- [x] text block is RTL and matches UX designs (except for service-specific font)
- [x] Font is bold
- [x] Font size is: 38px (<600px), 44px otherwise

**Testing notes**
* test fallback font on older devices 
* check with UX designers

**Testing URL's**
https://www.test.bbc.com/arabic/articles/c69dvq19k63o
https://www.test.bbc.com/persian/articles/c4vlle3q337o
https://www.test.bbc.com/urdu/articles/cx621klkm1ro
https://www.test.bbc.com/pashto/articles/cng0e8v85eko

Dev insight: Update relevant unit tests. 

**Additional context**

**Comment by sareh** [comment]
Blocked on adding Nassim Persian to Simorgh: https://github.com/bbc/simorgh/issues/1910

**Comment by jamesdonoh** [comment]
No longer blocked on #1910; now blocked on bbc/psammead#543

**Comment by jamesdonoh** [comment]
The above issue was closed, assuming this is now unblocked so we should try to pick up.

**Comment by jimjohnsonrollings** [comment]
I believe that other than the bolding everything for this is already done.

**Comment by jimjohnsonrollings** [comment]
Blocked pending #2732

**Comment by nataliesmart** [comment]
Headline (H1) type sizes:
Group A (0-319px): font size is 32px with line height 42px
Group B (320-599px): font size is 38px with line height 46px
Group D (600px +): font size is 44px with line height 54px

**Comment by jamesdonoh** [comment]
Unblocked following that bbc/psammead#1516

**Comment by RayNjeri** [comment]
Blocked pending #2713

**Comment by rhenshaw56** [comment]
I believe this could be worked on side by side with #2713 removing the blocked label

**Comment by RayNjeri** [comment]
> Headline (H1) type sizes:
> Group A (0-319px): font size is 32px with line height 42px
> Group B (320-599px): font size is 38px with line height 46px
> Group D (600px +): font size is 44px with line height 54px

Checked as per the description and per the values specified here by @nataliesmart and everything seems to check off, both on Chrome and Firefox. Further UX review is welcomed.
cc: @jamesdonoh 
![Screen Shot 2019-08-13 at 10 46 06 AM](https://user-images.githubusercontent.com/27012452/62924000-d31bf380-bdb7-11e9-9140-b56c5bde2fb7.png)

**Comment by RayNjeri** [comment]
Same applies on live.
<img width="1440" alt="Screen Shot 2019-08-13 at 11 44 49 AM" src="https://user-images.githubusercontent.com/27012452/62927723-cd2a1080-bdbf-11e9-8983-7964ce180b71.png">

**Comment by stevencava** [comment]
Headline block looks ok on all four services, font render issue of Urdu on mobile (This is a known issue on iOS)  @nataliesmart

**Comment by jimjohnsonrollings** [comment]
@stevencava What's the Urdu iOS issue?

**Comment by stevencava** [comment]
![Image from iOS (12)](https://user-images.githubusercontent.com/7226228/62934575-c8179200-bdbc-11e9-8ced-b9712a527722.png)

**Comment by jimjohnsonrollings** [comment]
Mine doesn't do that once Nassim has loaded.

**Comment by stevencava** [comment]
When I tap the refresh button it corrects itself

**Comment by jimjohnsonrollings** [comment]
Then I think we're happy with this :)

**Comment by stevencava** [comment]
Just noticed it seems to do it when off wifi

**Comment by jimjohnsonrollings** [comment]
What we're seeing as 'stretched' is the iOS default font, which will display if you've not managed to download Nassim quickly enough - from second page view Nassim should have successfully loaded.

**Comment by jamesdonoh** [comment]
Closing this issue as we believe the treatment on live is as expected, please reopen if not.

**Comment by nataliesmart** [comment]
I have also just checked this and all seems ok on my Chrome Version 76.0.3809.100. Thanks everyone.
